help/md/fnWAG.md

```diff
@@ -1,9 +1,27 @@
 ## name
 fnWAG
 ## title
+WAG (Whelan and Goldman) Empirical Amino Acid Substitution Model
 ## description
+The WAG rate matrix combines the strengths of likelihood-based and counting methods for estimating amino acid substitution patterns. 
+
 ## details
+The WAG model, a new empirical model of amino acid replacement derived using an approximate maximum-likelihood method from 3,905 sequences across 182 protein families. It outperforms widely used models like Dayhoff and JTT in terms of accuracy and likelihood for phylogenetic analysis, aiming to provide better evolutionary tree estimates and applications in sequence alignment, database searches, and protein structure prediction.
 ## authors
+
 ## see_also
+Dayhoff
+JTT
```

Comment:
I think we either call this model something else or straight up haven't implemented it. Please change or remove.

Reply:
We could also add `fnLG` as another "See also" item.